src/parsers/espresso/parser.js

```diff
@@ -0,0 +1,259 @@
+import { ATOMIC_COORD_UNITS, coefficients } from "@exabyte-io/code.js/dist/constants";
+
+import { ConstrainedBasis } from "../../basis/constrained_basis";
+import { primitiveCell } from "../../cell/primitive_cell";
+import { Lattice } from "../../lattice/lattice";
+import math from "../../math";
+import { MaterialParser } from "../init";
+import { FortranParser } from "../utils/fortran";
+import { IBRAV_TO_LATTICE_TYPE_MAP, regex } from "./settings";
+
+export class ESPRESSOMaterialParser extends MaterialParser {
```

Comment:
Reuse Fortran parser capability through a `FotranParserMixin` and multiple inheritance